src/TodoListItem.js

```diff
@@ -1,8 +1,11 @@
 import React from "react";
 
-function TodoListItem({ todoTitle }) { 
+function TodoListItem({ todoTitle, onRemoveTodo }) { 
   return (
-      <li>{todoTitle}</li>
+    <li>
+      {todoTitle}
+      <button type="button" onClick={() => onRemoveTodo()}>Remove</button>
+    </li>
```

Comment:
And this would be written like this

```suggestion
    <li>
      {todo.title}
      <button type="button" onClick={() => onRemoveTodo(todo.id)}>Remove</button>
    </li>
```

You don't _have_ to change anything because it does work, but this is a more common pattern for the flow of props.